vgen/src/main/java/io/gapi/vgen/csharp/CSharpMethodViewing.java

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gapi.vgen.csharp;
+
+import com.google.api.tools.framework.model.Method;
+import com.google.api.tools.framework.model.ProtoFile;
+
+import io.gapi.vgen.MethodViewing;
+
+public class CSharpMethodViewing extends MethodViewing
+    implements CSharpProtoElementViewing<Method> {
+
+  @Override
+  public ProtoFile getNamespaceFile(Method method) {
+    return method.getParent().getFile();
```

Comment:
I can live with that, but keep in mind that as a consequence, you are creating a new class in this PR and then deleting the class in the following PR, because the only reason this class exists is to provide getNamespaceFile. (So  I would argue that my suggestion reduces the scope... but I can't keep my mouth shut :-P )